yt/utilities/on_demand_imports.py

```diff
@@ -232,25 +232,6 @@ def __version__(self):
 _cartopy = cartopy_imports()
 
 
-class pooch_imports:
-    _name = "pooch"
-
-    _pooch = None
-
-    @property
-    def pooch(self):
-        if self._pooch is None:
-            try:
-                import pooch as pooch
-            except ImportError:
-                pooch = NotAModule(self._name)
-            self._pooch = pooch
-        return self._pooch
-
-
-_pooch = pooch_imports()
```

Comment:
Why was this deleted from on_demand_imports? We still need pooch here.

Reply:
indeed. In fact I don't remember doing this, this may be an rebasing accident !

Reply:
fixed !